### DISCUSSION TITLE
Streamlining product terms (e.g. Treasury, Bank, Vault, Minion, Safe Minion)

### DISCUSSION THREAD

**Discussion by arentweall** [odyssy-automaton/pokemol-web#1996] [Ideas]
Working on docs for the past year & dev docs more recently with @UI369, we saw 2 areas that could be simplified / streamlined in our UX writing, namely: 
1. Treasury vs Bank 
2. Vault, Minion vs Safe Minion 

I believe these were created at different points in time to support new features we were shipping. To a new DAOhaus user, these terms having great similarities with slight nuances may be a little confusing. 

Looking forward, we thought it will be helpful to streamline & simplify some of these terms after reviewing past usage of these features & future anticipation of the roadmap.

**Comment by plor** [comment]
For 1. I think Treasury should be the term used for this as Bank carries with it some negative connotations of the banking industry and such, but I'm open to other interpretations of these two terms.

For 2. I think Vault is the concept and there are different implementations that is worth distinguishing between.
 - The treasury is the vault held by the DAO, is ragequitable, and is implemented in the Moloch contract
 - The minion is an implementation (mostly no longer used?) where the contract holding funds is separate from but controlled by the DAO
 - Minion safe is an implementation where the contract holding funds is Gnosis Safe but a minion controlled by the DAO has been installed as a safe module.

Those are just off the top of my head, I'm open to correction or additional context. In terms of communication I'm not sure the best way to be clear about these distinctions, but I think we should avoid obscuring important details that might be important to the DAOs using these features.

**Reply by UI369** [reply]
Some thoughts on this: 

Replacing references to Bank with Treasury seems like an easy win. 

If we could just call all of the "money holding things" as "Vaults" that would be great - so we only have 2 terms, Treasury & Vaults.  But we probably still have to say "Safe Minion" - but if we said those are a type of Vault, then we have clear categories. 

We could say the Treasury is composed of Vaults, and includes the Treasury Vault, and then additional Vaults are Safe Minions.  Something like that. 

If there's no reason to make a minion that is not a safe minion, could that feature be hidden so we can reduce terms? Just have "Safe Minions"  (kinda helps the minion term make more sense in my opinion anyhow)

So 2 terms: Treasury & Vaults.   Then Treasury Vaults, and Safe Minion Vaults.
